shared/cfg/machines.cfg

```diff
@@ -46,3 +46,15 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
+    - s390:
+        only s390
```

Comment:
This is what works for me:
https://github.com/avocado-framework/avocado-vt/commit/94c9c2e6136d03eb6f3c941e4794fe1cd37a56f5